### ISSUE TITLE
CopyQ's window is blank

### ISSUE THREAD

**Issue by ooker777** [hluk/CopyQ#1549]
https://user-images.githubusercontent.com/6695517/103176769-49cf6900-48a7-11eb-9a9f-eb8de0d23dd1.mp4

I'm on Windows 10 version 20H2 build 19042.685

Note that it's N edition, which lacks some features. But I've installed [Media Feature Pack](https://support.microsoft.com/en-us/help/3145500/media-feature-pack-list-for-windows-n-editions "Media Feature Pack list for Windows N editions")

**Comment by hluk** [comment]
What does "N edition" mean?

What is the CopyQ version?

**Comment by ooker777** [comment]
The version Microsoft sells in Europe for legal requirement

https://www.howtogeek.com/322112/what-is-an-n-or-kn-edition-of-windows

**Comment by AO2233** [comment]
Same problem in 20H2 (OS version 19042.685) Professional Edition
CopyQ 3.13.0 latest
And it could even get crash.

**Comment by hluk** [comment]
Is there any error/warning in the logs? If you cannot get to the logs from the CopyQ with F12 shortcut, it should be possible to log to a separate file, e.g. to "MyDocuments" folder on Windows from PowerShell:

```powershell
$env:COPYQ_LOG_LEVEL = 'DEBUG'
$env:COPYQ_LOG_FILE = [Environment]::GetFolderPath("MyDocuments") + '\copyq.log'
echo "Logs will be written to $env:COPYQ_LOG_FILE"
& 'C:\Program Files (X86)\CopyQ\copyq.exe'
```

**Comment by AO2233** [comment]
@hluk 
A part of the log 
```
CopyQ ERROR [2021-01-16 22:34:19.622] :    gpg: error opening lockfile '/c/Users/ao/.gnupg/C:\Users\ao\AppData\Roaming\copyq\copyq.pub.lock': No such file or directory
CopyQ ERROR [2021-01-16 22:34:19.622] :    gpg: lockfile disappeared
```
The /.gnupg is exist in the path
![image](https://user-images.githubusercontent.com/59056753/104815123-b35ee980-584d-11eb-8015-e641935e122d.png)

The full log:
[copyq log.zip](https://github.com/hluk/CopyQ/files/5824564/copyq.log.zip)

And the way of 
>https://github.com/hluk/CopyQ/issues/1408
is not work for me. (T_T)

**Comment by hluk** [comment]
The gpg error looks like different problem, it shouldn't affect UI rendering, it could only fail to load encrypted tabs or decrypt an item.

**Comment by AO2233** [comment]
Yeah, I find you may be right.
In my situation, the number of times that it can't open the main windows (like issue #1408) is more than the window is blank.
(I use WSL1 and the gpg is link to WSL‘s gpg ?)
```
I reinstall it and turn off the encrypt option then it works well with no error.
Thank u 👍  o(*￣▽￣*)ブ
```

**Comment by AO2233** [comment]
Hi 😃
The blank window appears again, and it Ihappens very few times.😂
This time, I use F12 shortcut to get a error log, but the error happens in 22:56 (UTC +8), And the newest  error log is at 22:00 (UTC +8).
The characters (连接丢失) means lost connection.
![Snipaste_2021-01-29_22-57-49](https://user-images.githubusercontent.com/59056753/106291652-df835d00-6286-11eb-88d3-6d1ef12ef6bd.png)
And in the blank window, you can use the top menu (see the video) and the right-click menu.
![Snipaste_2021-01-29_22-55-47](https://user-images.githubusercontent.com/59056753/106292545-e9599000-6287-11eb-94ea-3416fbe38c82.png)

**Comment by AO2233** [comment]
Hi 😃
The blank window appears again, and it happens very few times.😂
This time, I use F12 shortcut to get a error log, but the error happens in 22:56 (UTC +8), And the newest  error log is at 22:00 (UTC +8).
The characters (连接丢失) means lost connection.
![Snipaste_2021-01-29_22-57-49](https://user-images.githubusercontent.com/59056753/106291652-df835d00-6286-11eb-88d3-6d1ef12ef6bd.png)
And in the blank window, you can use the top menu (see the video) and the right-click menu.
![Snipaste_2021-01-29_22-55-47](https://user-images.githubusercontent.com/59056753/106292545-e9599000-6287-11eb-94ea-3416fbe38c82.png)

**Comment by hluk** [comment]
Can you try a recent [development build](https://ci.appveyor.com/project/hluk/copyq/builds/37489079/artifacts)? It uses a newer version of Qt framework which could potentially fix the issue.

**Comment by AO2233** [comment]
👌perfect👌

**Comment by ultranity** [comment]
this problem can be fixed by #1689 , but current latest release (ver4.10) was built before it. 

looking forward to a release update 😃
